tools/project_manager/remote_file_manager.php

```diff
@@ -122,15 +122,9 @@
 $curr_abspath = "$uploads_dir/$curr_relpath";
 // The absolute path of the current directory.
 
-// $uploads_account is not required and may not be defined if they have not
-// set up FTP uploads. If it's blank, lets use a different string instead.
-// The point is to provide a "nice" representation of the absolute path
-// that doesn't reveal details of the upper reaches of the filesystem.
-if ($uploads_account) {
-    $curr_displaypath = "~$uploads_account/$curr_relpath";
-} else {
-    $curr_displaypath = "~uploads/$curr_relpath";
-}
+// Provide a "nice" representation of the absolute path that doesn't reveal
+// details of the upper reaches of the filesystem.
+$curr_displaypath = "~uploads/$curr_relpath";
```

Comment:
I just want to make sure that I'm understanding the code correctly. `$uploads_dir` is still `/home/dpscans` (at least on PROD), but the form on the project page will display `~uploads/` instead of `~dpscans/` after this change goes in?

Reply:
Correct, it's just a name -- nothing functionally changes because the code isn't using that name for anything except showing it to the user.

Reply:
Will need to be documented for squirrels not to pay any attention to what it says on the project page :D -- which on TEST is already the case, anyway.

Reply:
If it ends up being too confusing we an always hotfix the label in `pgdp-production`.